R/tibble.R

```diff
@@ -23,14 +23,19 @@
 #'   unquote-splice via [`!!!`].
 #' @param .rows The number of rows, useful to create a 0-column tibble or
 #'   just as an additional check.
-#' @param .tidy_names Treatment of invalid or duplicate column names:
-#'   - `NULL`: default, throw an error if there are any missing or duplicated names,
-#'   - `FALSE`: deliberately request a tibble with invalid names,
-#'   - `TRUE`: apply [tidy_names()] to the names,
-#'   - a function: apply custom name repair (e.g., `.tidy_names = make.names`
-#'     to get base R equivalence).
-#' @seealso [as_tibble()] to turn an existing list into
-#'   a data frame.
+#' @param .name_repair Treatment of problematic column names:
+#'   - `"none"`: Do nothing: no name repair, no name checking,
+#'   - `"valid"`: Eliminate missing or duplicated names,
+#'   - `"assert_valid"`: (default value), do not repair the names, but check they are `valid`,
+#'   - `"syntactic"`: Use `TODO: fill this in` to create `valid` and syntactic
```

Comment:
What's missing here?

Reply:
I've reworded now. If we had exported new name-fixers, I would list the relevant one here. But we are holding off on that.